kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -419,7 +419,7 @@ extension [A, S, E](effect: A < (Abort[E] & S))
         fl2: Flat[B],
         fr: Frame
     ): B < (Abort[Nothing] & S & S1) =
-        Abort.fold(onSuccess, onFail)(effect)
+        Abort.fold[E](onSuccess, onFail)(effect)
```

Comment:
The explicit type parameter here shouldn't be necessary. It seems the issue is the use of `Reducible`. Considering that `Abort[Nothing]` is included in `IO` and derived effects, I think we can remove `Reducible`. I'm planning to follow up on that. Let me know if you have concerns @hearnadam @johnhungerford

Reply:
When you say we can remove it, do you mean entirely or just from `Abort`? `Env` still uses `Reducible`, correct?

Reply:
Good point! I forgot about `Env`. It'd be only the `Abort`-related ones

Reply:
No luck removing `Reducible` from `Abort`. We need to be able to fully handle `Abort` in the APIs that handle panics anyway